development/metamaskbot-build-announce.js

```diff
@@ -14,6 +16,12 @@ function capitalizeFirstLetter(string) {
   return string.charAt(0).toUpperCase() + string.slice(1);
 }
 
+function executeGitCommand(command) {
+  return execSync(command)
+    .toString('utf8')
+    .replace(/[\n\r\s]+$/u, '');
```

Comment:
Perhaps trim() is enough here and more readable?

Reply:
good idea. modified 🙏